js/galleryfileaction.js

```diff
@@ -210,6 +216,13 @@ $(document).ready(function () {
 	var url = window.galleryFileAction.buildGalleryUrl('config', '', {extramediatypes: 1});
 	$.getJSON(url).then(function (config) {
 		window.galleryFileAction.buildFeaturesList(config.features);
+
+		if (config.mediatypes.length === 0) {
+			config.mediatypes = ['image/png', 'image/jpeg', 'image/gif', 'image/x-xbitmap', 'image/bmp'];
```

Comment:
You've made the mods before I had the time to reply :D. Since it's the browser which renders the BMPs directly, bypassing PHP, there shouldn't be an issue with that format. You can reset head ~1 if you want to keep it.